resources/js/pluralization.js

```diff
@@ -30,11 +30,17 @@ function pluralE (choice, choicesLength) {
   number = (number == 0) ? 0 : ((number == 1) ? 1 : ((number == 2) ? 2 : (((number % 100 >= 3) && (number % 100 <= 10)) ? 3 : (((number % 100 >= 11) && (number % 100 <= 99)) ? 4 : 5))));
   return Math.min(number, choicesLength - 1);
 }
+function pluralF (choice, choicesLength) {
+  let number = Math.abs(choice);
+  number = (number == 1) ? 0 : (((number == 2) ? 1 : ((number < 10 && number % 10 == 0)) ? 2 : 3);
```

Comment:
You have too much parenthesis:
```suggestion
  number = (number == 1) ? 0 : ((number == 2) ? 1 : (number < 10 && number % 10 == 0) ? 2 : 3);
```

Reply:
Sure, thanks, ready to merge?

Reply:
This is still unresolved.
You'll have to rebuild the assets too, but I can do it, with this fix too. However, you'll have to sign the CLA please.